images/ic_autorenew_white_24px.svg

```diff
@@ -0,0 +1,4 @@
+<svg fill="#FFFFFF" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These icons are from Google's material design icon set (https://design.google.com/icons/) which is free to use commercially. However I am not at all attached to them and if some wants to make custom icons for us that'd be awesome.